package.json

```diff
@@ -40,28 +40,34 @@
     "@types/react": "^18.0.0",
     "@types/react-native": "^0.69.1",
     "@types/react-test-renderer": "^18.0.0",
-    "babel-jest": "^29.0.0",
+    "babel-jest": "^29.0.2",
     "conventional-changelog-cli": "^2.0.11",
     "cp-cli": "^2.0.0",
     "dedent": "^0.7.0",
     "del-cli": "^5.0.0",
     "eslint": "^8.21.0",
     "flow-bin": "~0.170.0",
     "flow-copy-source": "^2.0.9",
-    "jest": "^29.0.0",
+    "jest": "^29.0.2",
     "react": "^18.0.0",
     "react-native": "~0.69.1",
     "react-test-renderer": "^18.0.0",
     "strip-ansi": "^6.0.0",
     "typescript": "^4.0.2"
   },
   "dependencies": {
-    "pretty-format": "^29.0.0"
+    "pretty-format": "^29.0.2"
   },
   "peerDependencies": {
     "react": ">=16.0.0",
     "react-native": ">=0.59",
-    "react-test-renderer": ">=16.0.0"
+    "react-test-renderer": ">=16.0.0",
+    "jest": ">=28.0.0"
+  },
+  "peerDependenciesMeta": {
+    "jest": {
+      "optional": true
```

Comment:
does this work with npm too? otherwise jest will be installed as a part of npm install

Reply:
Seems like yes: https://docs.npmjs.com/cli/v8/configuring-npm/package-json#peerdependenciesmeta

Reply:
FYI. npm 8 is available since node 16. node 14 and 15 has npm 7

Reply:
Npm v7 docs also mentions support for `peerDepsMeta` + `optional`: https://docs.npmjs.com/cli/v7/configuring-npm/package-json#peerdependenciesmeta